tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -47,6 +48,12 @@ const (
 	ruleAction = "Allow"
 	// azureUser used to access the windows instance
 	azureUser = "core"
+	// sshPort is the SSH port
+	sshPort = "22"
```

Comment:
Same comment here about sharing it between the test packages of AWS and Azure. You can have one pre-commit where you make it common for both test and regular packages.